docs/reference/field_types.rst

```diff
@@ -108,6 +108,37 @@ Parameter                               Description
         ;
     }
 
+``TemplateRegistry::TYPE_BOOLEAN``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can use the following parameters:
+
+======================================  ======================================================================
+Parameter                               Description
+======================================  ======================================================================
+**ajax_hidden**                         Yes/No; ajax_hidden allows to hide list field during an AJAX context.
+**editable**                            Yes/No; editable allows to edit directly from the list if authorized.
+**inverse**                             Yes/No; reverses the background color (green for false, red for true).
+======================================  ======================================================================
+
+.. code-block:: php
+
+    protected function configureListFields(ListMapper $listMapper)
+    {
+        // For the value `prog`, the displayed text is `In progress`. The `App` catalogue will be used to translate `In progress` message.
```

Comment:
I don' understand this comment. Is it respecting a previous example scenario or something like that?

Reply:
My bad, it was coming from a (bad) copy/paste.

IMHO, catalogue should be deprecated in favor of translation_domain.
But it's currently the valid option.

See https://github.com/sonata-project/SonataAdminBundle/issues/6116